alerting/src/main/kotlin/org/opensearch/alerting/workflow/CompositeWorkflowRunner.kt

```diff
@@ -310,7 +310,7 @@ object CompositeWorkflowRunner : WorkflowRunner() {
                 .should(boolQuery().mustNot(existsQuery(Alert.ERROR_MESSAGE_FIELD)))
                 .should(termsQuery(Alert.ERROR_MESSAGE_FIELD, ""))
             queryBuilder.must(noErrorQuery)
-            searchRequest.source().query(queryBuilder)
+            searchRequest.source().query(queryBuilder).size(9999)
```

Comment:
why?

Reply:
to make sure we fetch as many alerts as we can
we don't intend to evaluate with just 10 alerts which the default if no size is mentioned

Reply:
We should be paginating through them then. Also isnt there a max of 25 delegate monitors to a workflow?

Reply:
let's revisit this soon
https://github.com/opensearch-project/alerting/issues/1022 and take a quick follow up.